graylog2-web-interface/src/views/components/AdaptableQueryTabsConfiguration.tsx

```diff
@@ -145,7 +145,7 @@ const AdaptableQueryTabsConfiguration = ({ show, setShow, queriesList, dashboard
     <BootstrapModalConfirm showModal={show}
                            title="Update Dashboard Pages Configuration"
                            onConfirm={onConfirmPagesConfiguration}
-                           onModalClose={onPagesConfigurationModalClose}
+                           onCancel={onPagesConfigurationModalClose}
```

Comment:
We removed the `onModalClose` prop which is replaced with `onCancel`. Since the `BootstrapModal` api offer also `onModalClose` I would suggest keep in it in our component that way we can avoid this change. wha do you think?

Reply:
After we talked about it, we agreed on keeping onCancel.